### DISCUSSION TITLE
Integrate Winget into Microsoft Store

### DISCUSSION THREAD

**Discussion by ppvnf** [microsoft/winget-cli#2550] [General] [answered]
### Description of the new feature / enhancement

I was confused if I should suggest this here or through the feedback app, but I chose here since Winget already allows the use of msstore as a source. I know Microsoft is aware that MS Store needs more apps and is working towards allowing more than UWP there, like x86 apps. Meanwhile Winget is a much more recent endeavor and already has many useful packages available. Why can't we have the better of both products, the UI of Microsoft Store and the catalog of Winget? Why not make MS Store like Winget package + app description/info/pics + purchase options + comments?
Sorry if this is a stupid idea or if I am giving feedback in the wrong place.

**Comment by denelon** [comment, marked as answer]
Adding an application to the Microsoft Store requires a Partner Center account and a relationship with Microsoft. The decision to integrate the community repository as a source of packages from the Microsoft Store likely wouldn't be made by the WinGet team. We do work together frequently, so I will let them know this has been suggested.

Thank you for taking the time to submit the request.

If others are interested, please add your👍so we can track the interest.

**Comment by ppvnf** [comment]
I believe a first step would be to sync packages from MS Store and Winget. For example, if I installed PowerToys from MS Store, Winget should recognize that Microsoft.PowerToys Winget package is installed

**Comment by banzr** [comment]
I also would expect that `winget upgrade --all` should update apps installed from ms store. This is currently not consistent. For example, after running the upgrade command, I clicked `update all` in the ms store and the Photos app still needed to be updated.

**Comment by upintheairsheep** [comment]
I'm excited for this to happen, as it would allow more users and developers to flock to here and add whatever packages they want over here, and people would just actually use Microsoft Store opposed to "installer searching"  and filling your downloads folder with portable apps.

**Comment by YevheniiPokhvalii** [comment]
Is it somehow possible to sync the packages from Winget and Windows Store?
I'd like to automate installation and Winget seems like the best solution but at the same time all the packages I install via Winget do not sync with the Windows Store.

**Comment by upintheairsheep** [comment]
No, add WinGet as a separate tab from other MS Store apps and probably use
AI to merge some entries into their store versions.
